src/lib/dialog_ui.sh

```diff
@@ -241,6 +241,78 @@ function create_loading_screen_notification()
   run_dialog_command "$cmd" "$flag"
 }
 
+function spinFrame()
```

Comment:
Add simple documentation about this function. Also, use a camel case.

Reply:
Sorry, for intruding, but don't you mean `snake_case`?

Reply:
you are right, snake_case.